examples/node_prediction/citation_gcn.py

```diff
@@ -52,13 +59,13 @@
 model = Model(inputs=[x_in, a_in], outputs=gc_2)
 optimizer = Adam(lr=learning_rate)
 model.compile(optimizer=optimizer,
-              loss='categorical_crossentropy',
+              loss=CategoricalCrossentropy(reduction='sum'),
```

Comment:
'categorical_crossentropy' uses `AUTO` reduction, which defaults to `SUM_OVER_BATCH_SIZE`. AFAIK there is no way to use a weighted mean reduction, hence we normalize the weights and use `sum`.